warehouse/redshift/redshift.go

```diff
@@ -376,12 +381,33 @@ func (rs *HandleT) loadTable(tableName string, tableSchemaInUpload, tableSchemaA
 		partitionKey = column
 	}
 
+	var skipEntireTableClause string
+	if dedupWindow {
+		if _, ok := tableSchemaAfterUpload["received_at"]; ok {
+			skipEntireTableClause = fmt.Sprintf(`AND %[1]s.%[2]s.received_at > GETDATE() - INTERVAL '%d DAY'`, rs.Namespace, tableName, dedupWindowInDays)
+		}
```

Comment:
[minor] for readability It would be prefer  this statement to be after the:
```
		DELETE FROM 
		  %[1]s."%[2]s" using %[1]s."%[3]s" _source 
		where 
```

Keep appending on the main clause

Reply:
Make sense.

Reply:
Done.